File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/CSharp/Silverlight/1042/BusinessApplication/Assets/Resources/ApplicationResources.cs
namespace $safeprojectname$
{
    using System;
    using System.ComponentModel;
    using System.Globalization;
    using System.Windows.Browser;

    /// <summary>
    /// 컨트롤 속성을 XAML의 리소스 문자열에 바인딩할 수 있도록 강력한 형식의 리소스 클래스에 대한 액세스를 래핑합니다.
    /// </summary>
    public sealed class ApplicationResources
    {
        private static readonly ApplicationStrings applicationStrings = new ApplicationStrings();
        private static readonly ErrorResources errorResources = new ErrorResources();

        /// <summary>
        /// <see cref="ApplicationStrings"/>를 가져옵니다.
        /// </summary>
        public ApplicationStrings Strings
        {
            get { return applicationStrings; }
        }

        /// <summary>
        /// <see cref="ErrorResources"/>를 가져옵니다.
        /// </summary>
        public ErrorResources Errors
        {
            get { return errorResources; }
        }
    }
}

File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/VisualBasic/Silverlight/1042/BusinessApplication/Assets/Resources/ApplicationResources.vb
Imports System
Imports System.ComponentModel
Imports System.Globalization
Imports System.Windows.Browser


''' <summary>
''' 컨트롤 속성을 XAML의 리소스 문자열에 바인딩할 수 있도록 강력한 형식의 리소스 클래스에 대한 액세스를 래핑합니다.
''' </summary>
Public NotInheritable Class ApplicationResources
    Private Shared _applicationStrings As New ApplicationStrings()
    Private Shared _errorResources As New ErrorResources()

    ''' <summary>
    ''' <see cref="ApplicationStrings"/>를 가져옵니다.
    ''' </summary>
    Public ReadOnly Property Strings() As ApplicationStrings
        Get
            Return _applicationStrings
        End Get
    End Property

    ''' <summary>
    ''' <see cref="ErrorResources"/>를 가져옵니다.
    ''' </summary>
    Public ReadOnly Property Errors() As ErrorResources
        Get
            Return _errorResources
        End Get
    End Property
End Class
